src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -123,7 +123,12 @@ export default class SimidMapCreative extends BaseSimidCreative {
     returnToAdButton.onclick = () => this.playAd_(returnToAdButton); 
 
     const skipAdButton = document.createElement("button");
+    skipAdButton.id = "skipAd";
     skipAdButton.textContent = "Skip Ad";
+    const skipAdLogo = document.createElement("img");
+    skipAdLogo.id = "skipAdLogo";
+    skipAdLogo.src = "https://img.icons8.com/ios/100/000000/end.png";
```

Comment:
Where did you get this image from?  Make sure you're definitely cleared to use this and that the author has given permission.  It might be safer to create your own image if its not too complex of a design

Reply:
Or you can try creating one using svg for an extra challenge: https://www.w3schools.com/graphics/svg_intro.asp

Reply:
got rid of icon due to publishing issues